guide/payments/introduction.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: guide
+title: Payments
+nav_order: 3
+has_children: true
+main_classes: -no-top-padding
+permalink: /guide/payments/introduction/
+image: /assets/images/payments/introduction.png
+---
+
+{% include picture.html
+   image = "/assets/images/payments/introduction.png"
+   retina = "/assets/images/payments/introduction@2x.png"
+   mobile = "/assets/images/payments/introduction.png"
+   mobileRetina = "/assets/images/payments/introduction@2x.png"
+   alt-text = ""
+   width = 1600
+   height = 900
+   layout = "full-width"
+%}
+
+# Payments
+
+There are limitations in the current financial system that make payment services inaccessible to people in many places of the world — not everyone has access to bank accounts, identification, proof of address or are in countries which support the use of modern financial services.
+
+## Bitcoin is money
+
+> **Getting Started / Why bitcoin is unique**
+>
+> The economic definition of money typically involves three specific [functions](https://en.wikipedia.org/wiki/Money#Functions) (medium of exchange, unit of account, store of value) and five [properties](https://en.wikipedia.org/wiki/Money#Properties) (durability, portability, fungibility, scarcity, divisibility, recognizability). Although bitcoin is a new breed of money, it still satisfies all of these requirements. Instead of relying on physical properties (like gold and silver) or trust in central authorities (like traditional currencies), bitcoin relies on a network of thousands of computers to enforce its mathematically defined ruleset refered to as the protocol.
+
+## Bitcoin is Global
+
+We are for designing the next evolution of money. One which is internet based, and global.
+
+> **Getting Started / Why bitcoin is unique**
+>
+> Bitcoin can be used by anyone with an internet connection, which is available to around [53.6% of the world population](https://en.wikipedia.org/wiki/Global_Internet_usage), a number that is growing quickly. Some aspects of bitcoin, like key and address generation, even work without being online. This makes it the first truly globally available form of money that is unbound by country borders. People from around the world can freely transact without having to rely on middlemen like currency exchanges and local payment services.
+
+## Bitcoin is extensible, it's not just a payment network
+
+It should be noted, that bitcoin is not just a mechanism to make simple payments but it is a complete global financial infrasutructure for moving value without intermediaries. Each transaction is like a bit of code, which means it can be programmed — for example you can set various spending conditions like requiring multiple parties to agree prior funds being able to move. Its even possible to restrict funds from being moved until a certain time.
```

Comment:
Capital B, as I believe you're reoffering to the protocol here.
```suggestion
It should be noted, that Bitcoin is not just a mechanism to make simple payments but it is a complete global financial infrastructure for moving value without intermediaries. Each transaction is like a bit of code, which means it can be programmed — for example you can set various spending conditions like requiring multiple parties to agree prior funds being able to move. Its even possible to restrict funds from being moved until a certain time.
```

Reply:
Nice catch on the spelling. I think we settled to use lowercase bitcoin though, was part of "the first great debate" some months ago. Quite divisive lol

Reply:
But even in the guide it says that when you're reffering to the protocol it's capital B https://bitcoin.design/guide/getting-started/technology-primer/#is-it-bitcoin-or-bitcoin so would be good to implement what we're preaching 😃

Reply:
"Getting started" is based on the [Bitcoin Wiki](https://en.bitcoin.it/wiki/Help:Introduction):

> Since Bitcoin is both a currency and a protocol, capitalization can be confusing. Accepted practice is to use Bitcoin (singular with an upper case letter B) to label the protocol, software, and community, and bitcoins (with a lower case b) to label units of the currency.

I'd keep it simple and stick with that.

Reply:
Aha. Ok I interpreted it as lowercase everything.